<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compras</class>
 <widget class="QDialog" name="Compras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cupcake Store - Compras</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursoss.qrc">
    <normaloff>:/Imagens/imgs/ess.png</normaloff>:/Imagens/imgs/ess.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
	font-size: 16px;
	background: white;
}

QToolButton
{
	background: transparent;
	border-radius: 40px;
}

QPushButton
{
	background: pink;
	border-radius: 40px;
}

Qlabel
{
	color: pink;
	background: transparent;
}

QPushButton
{
	color: black;
	border-radius:15px;
}

QPushButton:hover
{
	color: #333;
	border-radius:15px;
	background: #49ebff;
}

QLineEdit
{
	background: pink;
	border: none;
	color: black;
	border-bottom: 1px solid white;
}

</string>
  </property>
  <widget class="QTableWidget" name="tab_listaProdutos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>561</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	border-color: pink;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_removerProduto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editarProduto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finalizarCompra">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar compra</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_valorTotal">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>391</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>R$</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tab_Produtos">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background: transparent;
	border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nossos produtos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>391</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor total:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="recursoss.qrc">
     <normaloff>:/Imagens/imgs/ess.png</normaloff>:/Imagens/imgs/ess.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pesquisarProduto">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>153</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CÃ³digo do produto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_codigo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>31</y>
     <width>101</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_qtde">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_codigo</tabstop>
  <tabstop>txt_qtde</tabstop>
  <tabstop>btn_pesquisarProduto</tabstop>
  <tabstop>tab_listaProdutos</tabstop>
  <tabstop>btn_removerProduto</tabstop>
  <tabstop>btn_editarProduto</tabstop>
  <tabstop>btn_finalizarCompra</tabstop>
  <tabstop>tab_Produtos</tabstop>
  <tabstop>toolButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="recursoss.qrc"/>
 </resources>
 <connections/>
</ui>
